source/features/add-dashboard-label-links.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {wrap} from '../libs/dom-utils';
+import observeEl from '../libs/simplified-element-observer';
+
+function linkifyLabels(activity: HTMLElement): void {
+	const labels = select.all('.IssueLabel', activity);
+	if (labels.length === 0) {
+		return;
+	}
+
+	const repository = select('a[data-hovercard-type="repository"]', activity) as HTMLAnchorElement;
+	if (!repository) {
+		return;
+	}
+
+	for (const label of labels) {
+		const search = new URLSearchParams();
+		const text = (label.textContent || '').trim();
+		search.set('q', `is:issue is:open sort:updated-desc label:"${text}"`);
+		wrap(label, <a href={`${repository.href}/issues?${search}`} />);
+	}
+}
+
+function findActivities(): void {
```

Comment:
`findActivities` receives the `observer` as an argument; you can use it to disable it altogether instead of using `rgh-linkified-labels`:

```js
observer.disconnect();
```